### PR TITLE
Relays terminate QUIC connections

### PR DESCRIPTION
**Pull request by afrind**
Fixes: #336

### CODE REVIEW

File: draft-ietf-moq-transport.md [hunk 1]
```diff
@@ -683,6 +683,9 @@ similar in functionality to Content Delivery Networks
 (CDNs). Additionally, relays serve as policy enforcement points by
 validating subscribe and publish requests at the edge of a network.
 
+Relays are MoQ endpoints, which means they terminate the underlying QUIC
+connections in order to have visibility of MoQ Object metadata.
```
Comment (ianswett) [inline draft-ietf-moq-transport.md:687]:
```suggestion
Relays are MoQ endpoints, which means they terminate the MoQ session
in order to have visibility of MoQ Object metadata.
```

**Review by ianswett — APPROVED** [review]
LG, but given Webtransport is also a substrate MoQ runs over, I think we want to avoid using QUIC connection?

**Review by suhasHere — APPROVED** [review]

**Review by fluffy — APPROVED** [review]
LGTM

**Review by ianswett — APPROVED** [review]
LG, but given Webtransport is also a substrate MoQ runs over, I think we want to avoid using QUIC connection?

**Pull request merged**